packages/user-operation-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.1.0]
+
+### Changed
+
+- Enable `resetMocks` Jest configuration option ([#4417](https://github.com/MetaMask/core/pull/4417))
```

Comment:
This change is not visible to consumers, so we can leave it out.

That said, we should document the `transaction-controller` bump:

``` suggestion
- Bump `@metamask/transaction-controller` to `^33.1.0` ([#4460](https://github.com/MetaMask/core/pull/4460))
```